src/main/java/knightminer/inspirations/building/block/BlockFlower.java

```diff
@@ -1,157 +1,98 @@
 package knightminer.inspirations.building.block;
 
-import java.util.Locale;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Random;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-
-import net.minecraft.block.Block;
-import net.minecraft.block.BlockBush;
-import net.minecraft.block.BlockDoublePlant.EnumPlantType;
-import net.minecraft.block.IGrowable;
-import net.minecraft.block.SoundType;
-import net.minecraft.block.properties.PropertyEnum;
-import net.minecraft.block.state.BlockStateContainer;
-import net.minecraft.block.state.IBlockState;
-import net.minecraft.creativetab.CreativeTabs;
-import net.minecraft.init.Blocks;
+import knightminer.inspirations.common.Config;
+import net.minecraft.block.*;
+import net.minecraft.block.material.Material;
+import net.minecraft.item.ItemGroup;
 import net.minecraft.item.ItemStack;
-import net.minecraft.util.IStringSerializable;
 import net.minecraft.util.NonNullList;
-import net.minecraft.util.math.AxisAlignedBB;
 import net.minecraft.util.math.BlockPos;
-import net.minecraft.world.IBlockAccess;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.util.math.shapes.ISelectionContext;
+import net.minecraft.util.math.shapes.VoxelShape;
+import net.minecraft.util.math.shapes.VoxelShapes;
+import net.minecraft.world.IBlockReader;
 import net.minecraft.world.World;
 
-public class BlockFlower extends BlockBush implements IGrowable {
-	public static final PropertyEnum<FlowerType> TYPE = PropertyEnum.create("type", FlowerType.class);
-
-	public BlockFlower() {
-		this.setDefaultState(this.blockState.getBaseState().withProperty(TYPE, FlowerType.ROSE));
-		this.setHardness(0F);
-		this.setSoundType(SoundType.PLANT);
-	}
-
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
-	/* Blockstate */
+public class BlockFlower extends BushBlock implements IGrowable {
```

Comment:
Actually we can't, since the vanilla block requires a potion effect for [Suspicious Stew](https://minecraft.gamepedia.com/Suspicious_Stew), though we probably should/could add that. `getCollisionShape()` is redundant since I forgot `doesNotBlockMovement()`, but everything else is specific for the double-height growing ability.